src/voice-broadcast/models/VoiceBroadcastPlayback.ts

```diff
@@ -210,7 +256,7 @@ export class VoiceBroadcastPlayback
     private async tryLoadPlayback(chunkEvent: MatrixEvent): Promise<void> {
         try {
             return await this.loadPlayback(chunkEvent);
-        } catch (err) {
+        } catch (err: any) {
```

Comment:
Feel free to correct me here, but I think that the better way to do this is to declare this as unknown here, then do something like `if(err instanceOf Error) { ...do logging etc }`.

I think I'm right in saying this is what TS strict mode does implicitly when turned on, but don't quote me on that.

Reply:
Here it should be logged and set to error state anyway. `message` will them be undefined :man_shrugging: